apis/management.cattle.io/v3/k8s_windows_default.go

```diff
@@ -43,26 +43,30 @@ var (
 		"v1.12": {
 			Kubelet: map[string]string{
 				"tls-cipher-suites":        "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
-				"feature-gates":            "MountPropagation=false,HyperVContainer=true",
+				"feature-gates":            "HyperVContainer=true",
 				"cgroups-per-qos":          "false",
 				"enforce-node-allocatable": "",
 				"resolv-conf":              "",
-				"cadvisor-port":            "",
 			},
 		},
 		"v1.13": {
 			Kubelet: map[string]string{
 				"tls-cipher-suites":        "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
-				"feature-gates":            "MountPropagation=false,HyperVContainer=true",
+				"feature-gates":            "HyperVContainer=true",
 				"cgroups-per-qos":          "false",
 				"enforce-node-allocatable": "",
 				"resolv-conf":              "",
 			},
 		},
+		"v1.14": {
+			Kubelet: map[string]string{
+				"tls-cipher-suites": "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
```

Comment:
Is there only one parameters `tls-cipher-suites` for kubelet parameters?
I see that we have multiple parameters in `v1.13`

Reply:
updated